pyproject.toml

```diff
@@ -30,7 +30,9 @@ exclude = [
     # Exclude anything in an examples directory, except the docs/examples dir.
     '^(?!docs\/examples\/).*examples\/.*$',
     # Exclude our test files where we invoce mypy from pytest.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick (non-blocking):** ‏ There's a typo in this comment that we could fix here -> `invoce` to `invoke`

Reply:
Fixed in 78c05ea